estate/controllers/main.py

```diff
@@ -0,0 +1,26 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.http import Controller
+from odoo.http import request 
+from odoo.http import route 
```

Comment:
```suggestion
from odoo.http import Controller, request, route
```
Need of writing this way? :thinking: